app/dashboard/tests/test_dashboard_helpers.py

```diff
@@ -50,3 +55,28 @@ def test_amount(self):
     def test_normalize_url(self):
         """Test the dashboard helper normalize_url method."""
         assert normalize_url('https://gitcoin.co/') == 'https://gitcoin.co'
+    
+    @patch('marketing.mails.featured_funded_bounty')
+    def test_featured_funded_mail(self, mock_featured_funded_bounty):
+        """Test sending featured funded bounty mail."""
+        Bounty.objects.create(
+            title='foo',
+            value_in_token=3,
+            token_name='ETH',
+            web3_created=datetime(2008, 10, 31, tzinfo=pytz.UTC),
+            github_url='https://github.com/gitcoinco/web/issues/305#issuecomment-999999999',
+            token_address='0x0',
+            issue_description='hello world',
+            bounty_owner_github_username='iamonuwa',
+            bounty_owner_email='matrix4u2002@gmail.com',
+            is_open=False,
+            accepted=False,
+            expires_date=datetime(2008, 11, 30, tzinfo=pytz.UTC),
+            idx_project_length=5,
+            project_length='Months',
+            bounty_type='Feature',
+            experience_level='Intermediate',
+            raw_data={},
+            is_featured=True
+        )
+        assert mock_featured_funded_bounty().call_count == 0
```

Comment:
So in this test, we are confirming that the featured bounty email is sent - but here, the `call_count` is 0, indicating that an email was not sent. This test needs to assert that `call_count == 1`, which shows that an email would have been sent.

Reply:
Got that. I'll refactor this now.